package.json

```diff
@@ -3,9 +3,9 @@
   "name": "@zendeskgarden/react-components",
   "version": "0.0.0",
   "scripts": {
-    "build": "lerna run build --stream",
+    "build": "lerna run build --stream --concurrency 3",
```

Comment:
Rollup is overall faster than webpack, but the default concurrency of "all the packages" is really bogging it down. Building 3 packages at a time seems like the sweet spot of `~55s`